dmutils/filters.py

```diff
@@ -3,3 +3,13 @@
 
 def markdown_filter(text, *args, **kwargs):
     return markdown(text, ['markdown.extensions.abbr'], *args, **kwargs)
+
+
+def smart_join_filter(list_to_join):
+    lower_case = list(map(lambda text: text.lower(), list_to_join))
+    if len(lower_case) > 1:
+        return '{} and {}.'.format(', '.join(lower_case[:-1]), lower_case[-1])
```

Comment:
I'd skip the final dot here. It can always be added in the parent template and without it the filter can be used in the middle of a sentence.